style.css

```diff
@@ -1,3 +1,116 @@
-body h1{
+body, html {
+  margin: 0;
+  padding: 0;
+}
+
+body h1 {
   color: red;
+}
+
+.klasa {
+  width: 100px;
+  height: 100px;
+  box-shadow: 0 2px 4px 4px;
+}
+
+.container {
+  display: flex;
+  justify-content: space-around;
+}
+
+#klasa1 {
+  background-color: rgb(255, 0, 0);
+}
+
+#klasa2 {
+  background-color: rgb(0, 255, 0);
+}
+
+#klasa3 {
+  background-color: rgb(0, 0, 255);
+}
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child
In task it was specified that you don't use different class names, that also meant not to style it with different id's